UnityGLTF/Assets/UnityGLTF/Runtime/Shaders/PbrMetallicRoughness.shader

```diff
@@ -80,6 +82,21 @@ Shader "GLTF/PbrMetallicRoughness"
 			ENDCG
 		}
 		// ------------------------------------------------------------------
+		//  Vertex color pass
+		Pass
+		{
```

Comment:
Does this need to be its own pass?  I had thought it would just be some math/logic added to the existing pass.